recipes/phonenumbers/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "7.4.1" %}
+
+package:
+  name: phonenumbers
+  version: {{ version }}
+
+source:
+  fn: phonenumbers-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/source/p/phonenumbers/phonenumbers-{{ version }}.tar.gz
```

Comment:
I think we are preferring `https://pypi.io/` rather than `https://files.pythonhosted.org`